image/image.go

```diff
@@ -202,6 +202,18 @@ func bootstrapToRootDir(sto Store, model *asserts.Model, opts *Options) error {
 			return err
 		}
 
+		// fetch the snap assertions too
```

Comment:
even if it's just  a few lines right now I would still split it out in some kind of helper

fetchSnapAssertions that takes a reasonable set of args